import pandas as pd
import seaborn as sns
df =  pd.read_csv('spotify2023.csv', encoding = "ISO-8859-1")

df["in_apple_charts"] = pd.to_numeric(df["in_apple_charts"], errors="coerce")
sns.histplot(data=df, x="in_apple_charts", bins=50)


In [4]:
Max = df["in_apple_charts"].max()
Q3 = df["in_apple_charts"].quantile(.75)
Median = df["in_apple_charts"].median()
Q1 = df["in_apple_charts"].quantile(.25)
Min = df["in_apple_charts"].min()

summary = {
    "max" : Max,
    "Q3" : Q3,
    "Median":Median,
    "Q1" : Q1,
    "Min": Min
}

summary

{'max': 275, 'Q3': 87.0, 'Median': 38.0, 'Q1': 7.0, 'Min': 0}

#

# the group of plays in the apple chart that is the max is 275, the Quartile 3 is 87.0, the median is 38.0, the Quartile 1 is 7.0, and the minimum is 0